js/wavesexchange.js

```diff
@@ -32,6 +32,9 @@ module.exports = class wavesexchange extends Exchange {
                 'fetchOrders': true,
                 'fetchTicker': true,
                 'fetchTrades': true,
+                'getMatcherPublicKey': true,
```

Comment:
not sure if this should be in here

Reply:
Not all methods should be there, some of them are just exchange-specific helpers.